README.md

```diff
@@ -197,7 +204,8 @@ When using the custom server CA, the CA needs to be used for
 the server certificates on the capsules as well. The certificates for
 the capsule are deployed to the capsule through the use of the
 `capsule-certs-generate` script (followed by copying the certs tar to
-the capsule and running the capsule-installer to refresh the certificates).:
+the capsule and running the 'foreman-installer --scenario capsule'
+to refresh the certificates).:
```

Comment:
shouldn't we mention 'foreman-installer --scenario capsule <params>' or something similar. Now with the --scenario parameter it looks it is all that is needed.